poucave/html/js/app/components/CheckDetails.mjs

```diff
@@ -0,0 +1,277 @@
+    `));
+      parameters = html`<dl>${parameterItems}</dl>`;
+    }
+
+    let duration = null;
+    if (result.duration) {
+      duration = html`
+      <span class="float-right text-gray-medium lh-1">
+        Executed in ${result.duration}ms.
+      </span>
+    `;
+    }
+
+    const updated = result.datetime ? new Date(result.datetime) : new Date();
+    let resultData = null;
+    if (result.data) {
+      resultData = html`
+      <h4>Result Data</h4>
+      <pre class="mb-0 check-result">${JSON.stringify(result.data, null, 2)}</pre>
+      <div class="mt-2 mb-3 text-gray-medium">
+        Updated <${TimeAgo} date="${updated}" />
+      </div>
+    `;
+    }
+
+    let plot = html`<em>No history.</em>`;
+    if (result.history?.length > 0) {
+      plot = html`<div id="${this.plotDivID}" />`;
+    }
+
+    let bugList = html`<em>No bugs.</em>`;
+    if (result.buglist && result.buglist.length) {
+      bugList = html`
+      <ul>
+        ${result.buglist.map(
+        (bug) => html`<li class="${bug.open ? "open" : "closed"} ${bug.heat}">
+            <a
+              title="${bug.status} - ${bug.summary} (updated: ${bug.last_update})"
+              href="${bug.url}"
+              target="_blank"
+            >
+              ${bug.id}
+            </a>
+            ${" ("}${bug.status}${", "}${timeago().format(bug.last_update)})
+          </li>`
+      )}
+      </ul>
+    `;
+    }
+
+    let statusClass = "text-gray";
+    let icon = "question-circle"
+    if (!result.isLoading) {
+      statusClass = result.success ? "text-green" : "text-red";
+      icon = result.success ? "check-circle" : "times-circle";
+    }
+
+    return html`
+      <div class="check-details slideout-panel ${opened ? "open" : ""}">
+        <div class="close-button" onClick=${onClickCloseButton} />
+        <div class="slideout-panel-content">
+          <div class="d-flex align-items-center">
+            <h2 class="d-flex align-items-center flex-grow-1">
+              <i class="fa fa-${icon} ${statusClass}" />
+              <span class="ml-2 flex-grow-1">${data.project}/${data.name}</span>
+            </h2>
+            <span class="text-right">
+              <button
+                class="check-refresh btn btn-sm btn-secondary"
+                onClick="${this.handleRefreshButtonClick}"
+                disabled="${result.isLoading}"
+              >
+                <i class="fa fa-sync-alt mr-1 ${result.isLoading ? 'fa-spin' : ''}"></i>
+                Refresh
+              </button>
+            </span>
+          </div>
+          
+          <hr />
+          
+          <div class="w-50">
+            ${tags}
+            <${Markdown} md="${data.description}" />
+          </div>
+          
+          <hr />
+          
+          <div class="d-flex">
+            <div class="w-50">
+              <h4>Documentation</h4>
+              <${Markdown} md="${data.documentation}" />
+            </div>
+            <div class="pl-2">
```

Comment:
Shouldn't it be `w-50` here too?

Reply:
because of flex this will just fill the available space